projects/android/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea
+.DS_Store
+build/
+/captures
+.externalNativeBuild
+.cxx
```

Comment:
Are we ignoring all C++ files?

Reply:
Interesting. I'm unsure why that's there. This file was generated by Gradle.

Reply:
Removed that line (and merged this .gitignore into the root file)